models/src/node/mod.rs

```diff
@@ -188,6 +188,17 @@ impl BottlerocketNodeSpec {
         // We know this won't panic because we have a regex requirement on this attribute, which is enforced by the k8s schema.
         self.version.as_ref().map(|v| Version::from_str(v).unwrap())
     }
+
+    /// Returns the operation for current spec.
+    pub fn operation(&self) -> String {
```

Comment:
Consider using [`serde_plain`](https://crates.io/crates/serde_plain) to serialize these. We also use this library in the Bottlerocket repo for enum serialization, e.g. in [buildsys](https://github.com/bottlerocket-os/bottlerocket/blob/00ecda47174e3867e5549ced93579fc2a886eadb/tools/buildsys/src/main.rs#L112)